frontend/packages/ux-editor/src/components/config/expressions/DynamicContent.tsx

```diff
@@ -1,14 +1,16 @@
 import React, { useEffect, useRef } from 'react';
 import {
+  Dynamic,
+  ExpressionElement,
   expressionFunctionTexts,
   expressionInPreviewPropertyTexts,
   ExpressionPropertyBase,
-  expressionPropertyTexts
+  expressionPropertyTexts,
+  Operator
 } from '../../../types/Expressions';
-import { Button, ButtonColor, ButtonVariant, Select } from '@digdir/design-system-react';
-import { XMarkIcon, PencilIcon, ArrowRightIcon } from '@navikt/aksel-icons';
-import { ExpressionContent, ExpressionElement } from './ExpressionContent';
-import { Dynamic } from '../../rightMenu/Dynamics';
+import { Alert, Button, ButtonColor, ButtonVariant, Select } from '@digdir/design-system-react';
```

Comment:
In today's release of `digdir/designsystem` this enum is removed. You can just pass a plain string into button-colour and button-variant instead of using the enums. :) VSCode should give you IntelliSense on available string values. 

```suggestion
import { Alert, Button, Select } from '@digdir/design-system-react';
```